pkg/workload/schemachange/schemachange.go

```diff
@@ -1049,13 +1030,16 @@ ORDER BY random()
 }
 
 // randTable returns a schema name along with a table name
-func (w *schemaChangeWorker) randTable(tx *pgx.Tx, pctExisting int) (string, string, error) {
+func (w *schemaChangeWorker) randTable(tx *pgx.Tx, pctExisting int) (tree.TableName, error) {
 	if w.rng.Intn(100) >= pctExisting {
-		randSchema, err := w.randSchema(tx, pctExisting)
+		randSchema, err := w.randSchema(tx, 90)
```

Comment:
pctExisting` will be a low value when `randTable` is called with the intention of creating a table, so a non existing table is returned most of the time. Before I added schemas, this means that a create table operation would mostly succeed because most of the time a non existing table name is returned. However, when you pass `pctExisting` to rand schema, the schema will most likely not exist. This would cause the create table operation to fail most of the time. To restore the original behaviour of normally succeeding, I thought 90 would be a good value here. 

The same thing applies to createEnum and createView.